src/components/Map/Map.tsx

```diff
@@ -39,27 +46,27 @@ function Map({
         <Legend hideLegendTitle={hideLegendTitle}>
           <LegendItem
             key={'legend-5'}
-            title={LOCATION_SUMMARY_LEVELS[Level.SUPER_CRITICAL].summary}
+            title={LOCATION_SUMMARY_LEVELS[Level.SUPER_CRITICAL].summary || ''}
```

Comment:
FWIW- I'm not sure why `summary` is optional on `LevelInfoMap`.  I think it should always be defined. 🤔   You could explore fixing that if you felt inclined, but also okay to skip that rabbit hole for now. 😁

Reply:
It *is* always defined, but because of how the types are set up, typescript can't deduce that (I think).  This would be nice to follow up with better indexing or an enum or something, but I didn't want this work to get stuck on that.